code/modules/research/designs/misc_designs.dm

```diff
@@ -334,6 +334,76 @@
 	category = list("Equipment")
 	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING | DEPARTMENTAL_FLAG_SECURITY | DEPARTMENTAL_FLAG_SCIENCE
 
+/datum/design/dice_bag
+	name = "Bag of Dice"
+	desc = "An assorted bag of colorful plastic dice."
+	id = "dice_bag"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(/datum/material/plastic = 1000)
+	build_path = /obj/item/storage/dice
+	category = list("Equipment")
+	departmental_flags = DEPARTMENTAL_FLAG_SERVICE
+
+/datum/design/card_deck
+	name = "Deck of Cards"
+	desc = "A deck of NT standard playing cards."
+	id = "card_deck"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(/datum/material/plastic = 1000)
+	build_path = /obj/item/toy/cards/deck
+	category = list("Equipment")
+	departmental_flags = DEPARTMENTAL_FLAG_SERVICE
+
+/datum/design/crayon_box
+	name = "Box of Crayons"
+	desc = "A box full of crayons"
+	id = "crayon_box"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(/datum/material/plastic = 1000)
+	build_path = /obj/item/storage/crayons
+	category = list("Equipment")
+	departmental_flags = DEPARTMENTAL_FLAG_SERVICE
+
+/datum/design/piano_synth
+	name = "Synthesizer"
+	desc = "An advanced electronic synthesizer that can be used as various instruments."
+	id = "piano_synth"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(/datum/material/iron = 1000, /datum/material/silver = 200)
+	build_path = /obj/item/instrument/piano_synth
+	category = list("Equipment")
+	departmental_flags = DEPARTMENTAL_FLAG_SERVICE
+
+/datum/design/cutout
+	name = "Cardboard Cutout"
+	desc = "A vaguley humanoid sheet of synthetic cardboard. Can be customized with a crayon."
+	id = "cardboard_cutout"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(/datum/material/plastic = 2000)
+	build_path = /obj/item/cardboard_cutout
+	category = list("Equipment")
+	departmental_flags = DEPARTMENTAL_FLAG_SERVICE
```

Comment:
Plastic doesn't make sense for cardboard cutouts either maybe make the techlathe accept cardboard as well(making it an acceptable mat datum)

Reply:
Yeah i was thinking something like synthetic cardboard, mainly to explain how it can be printed from a lathe. To be honest i wouldn't be against having cardboard usable in lathes, but is it okay to have a lathe material that's only used in two recipes?

Reply:
it hath been done